core/import.go

```diff
@@ -160,7 +160,7 @@ listingLoop:
 				errChan <- fmt.Errorf("Error in record %d: %s", i, "price is a mandatory field")
 				return
 			}
-			if NormalizeCurrencyCode(listing.Metadata.PricingCurrency) != "BTC" {
```

Comment:
This command is going to need to be fixed at some point. It was never updated since we allowed multiple currencies.

Reply:
@cpacia Can you describe what change needs to be made in a new issue so we don't lose it?